packages/katello/katello/backup.rb

```diff
@@ -324,7 +324,14 @@ def generate_metadata
     end
 
     def create_pulp_data_tar
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      @archive_size = @options[:split_pulp_tar] || "0"
+      @split_pulp_tar_script = default_split_pulp_tar_script
+
+      if @archive_size == "0"
+        run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      else
+        run_cmd("tar --selinux --create --tape-length=#{@split_size} --new-volume-script=#{@split_pulp_tar_script} --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
```

Comment:
this and line 331 repeats a lot of the tar command. You could build the tar command dynamically to DRY it up i.e.
```ruby
tar_command = "tar --selinux --create "
tar_command += "--tape-length=#{archive_size} --new-volume-script=#{split_pulp_tar_script} " if archive_size
tar_command += "--exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *"
run_cmd(tar_command)
```

Reply:
Also note I had to change `@split_size` to `archive_size` as I don't see that defined anywhere

Reply:
Good call, I'll get that fixed